platform/platform-migratable/src/test/java/org/codice/ddf/platform/migratable/impl/PlatformMigratableTest.java

```diff
@@ -0,0 +1,273 @@
+
+    private static final String DESCRIPTION = "Exports system files";
+
+    private static final boolean IS_OPTIONAL = false;
+
+    private Path ddfHome;
+
+    private Path exportDirectory;
+
+    @Before
+    public void setup() throws Exception {
+        ddfHome = Paths.get(DDF_BASE_DIR);
+        exportDirectory = ddfHome.resolve(EXPORTED_DIR_REL_PATH);
+    }
+
+    @Test
+    public void testExportValidRelativePaths() throws Exception {
+        // Setup
+        MigratableUtil mockMigratableUtil = mock(MigratableUtil.class);
+        PlatformMigratable platformMigratable = new PlatformMigratable(DESCRIPTION, IS_OPTIONAL,
+                mockMigratableUtil);
+
+        // Perform Test
+        platformMigratable.export(exportDirectory);
+
+        // Verify ws-security directory is exported
+        assertWsSecurityExport(mockMigratableUtil);
+
+        // Verify system.properties and users.properties are exported
+        assertSystemPropertiesFilesExport(mockMigratableUtil);
+
+        // Verify keystore and truststore are exported
+        assertKeystoresExport(mockMigratableUtil);
+    }
+
+    /**
+     * Verify that if an absolute path is encountered during the export, a warning is returned.
+     */
+    @Test
+    public void testExportWarningsReturnedWhenExportingKeystoreAndTrustsore() throws Exception {
+        // Setup
+        MigratableUtil mockMigratableUtil = mock(MigratableUtil.class);
+        MigrationWarning expectedMigrationWarning1 = new MigrationWarning("warning 1");
+        doAnswer(new Answer<Void>() {
+            @SuppressWarnings("unchecked")
+            public Void answer(InvocationOnMock invocation) {
+                Object[] args = invocation.getArguments();
+                ((Collection<MigrationWarning>) args[2]).add(expectedMigrationWarning1);
+                return null;
+            }
+        }).when(mockMigratableUtil).copyFileFromSystemPropertyValue(eq(KEYSTORE_SYSTEM_PROP), eq(exportDirectory), Matchers.<Collection<MigrationWarning>> any());
+        MigrationWarning expectedMigrationWarning2 = new MigrationWarning("warning 2");
+        doAnswer(new Answer<Void>() {
+            @SuppressWarnings("unchecked")
+            public Void answer(InvocationOnMock invocation) {
+                Object[] args = invocation.getArguments();
+                ((Collection<MigrationWarning>) args[2]).add(expectedMigrationWarning2);
+                return null;
+            }
+        }).when(mockMigratableUtil).copyFileFromSystemPropertyValue(eq(TRUSTSTORE_SYSTEM_PROP), eq(exportDirectory), Matchers.<Collection<MigrationWarning>> any());
+
+        PlatformMigratable platformMigratable = new PlatformMigratable(DESCRIPTION, IS_OPTIONAL,
+                mockMigratableUtil);
+
+        // Perform Test
+        MigrationMetadata migrationMetadata = platformMigratable.export(exportDirectory);
+
+        // Verify
+        assertThat(migrationMetadata.getMigrationWarnings().size(), is(2));
```

Comment:
`containsInAnyOrder()` already checks the size so we shouldn't need this assertion.